src-graphql/client.js

```diff
@@ -0,0 +1,13 @@
+import GraphQLJSClient from '@shopify/graphql-js-client';
```

Comment:
How do you feel if, for the time being, we keep it just for the sake of being consistent, then figure out what to actually name this library?